web/js/naturalEvents/request.js

```diff
@@ -1,17 +1,18 @@
-var wv = wv || {};
-wv.naturalEvents = wv.naturalEvents || {};
-/**
- * @module wv.naturalEvents.request
- */
-wv.naturalEvents.request = wv.naturalEvents.request || function (models, ui, config) {
+import $ from 'jquery';
+import lodashOrderBy from 'lodash/orderby';
+import lodashUniqBy from 'lodash/uniqby';
+
+import util from '../util/util';
+
+export default function naturalEventsRequest (models, ui, config) {
   var self = {};
-  self.events = wv.util.events();
+  self.events = util.events();
 
   self.EVENT_QUERY_RESULTS = 'queryResults';
   self.EVENT_SELECT = 'select';
 
   self.apiURL = config.features.naturalEvents.host;
-  var querySuccessFlag = false;
+  self.querySuccessFlag = false;
```

Comment:
Why is this set to `self.querySuccessFlag` now?

Reply:
The linter was complaining that `querySuccessFlag` wasn't being used, even though it's used around line 26/27. Attaching it to the `self` object instead fixes the linter error and behaves the same way.